tests/test_contrib_to_marc21_utils.py

```diff
@@ -55,3 +56,21 @@ def test_entry_points():
     output = dump(data,
                   xslt_filename='{0}/demo_marc21_to_dc.xslt'.format(path))
     assert output.decode('utf-8') == expect
+
+
+def test_output_type_from_dumps_etree():
```

Comment:
More important is to test that `dumps` with `xslt_filename` returns correct `xml` output.